### DISCUSSION TITLE
Max image filesize with Image thumbnails

### DISCUSSION THREAD

**Discussion by Hbitvof** [pimcore/pimcore#7821] [Q&A / Support]
Hi,

Is there a way to set maximum filesize with image thumbnails configuration? On many file editor tools you can set maximum filesize when saving a file?

**Comment by brusch** [comment]
No this is currently not possible, because neither Imagick nor GD offers a generic method that supports all (or even most) target image formats (ImageMagick only support `jpeg` as far as I'm aware).

**Comment by Hbitvof** [comment]
@brusch we want to use it for jpeg. It's the format for our website where we only use jpeg as format for our product images.

With Imagick this is possible with parameter -define jpeg:extent=max_value.

Is it possible to add this as option to the thumbnail configuration transformations?

**Comment by Hbitvof** [comment]
@brusch can you please confirm this is possible to implement?
